typescript/organisations.d.ts

```diff
@@ -63,6 +63,7 @@ export type TierLimitation =
 export interface NewTier {
   name: string
   tierData: {
+    maximumForms?: TierLimitation
```

Comment:
Why is it undefined?

Reply:
It does exist in production, there will be tiers that do not have this.